spec/iot-js-spec.md

```diff
@@ -183,23 +171,21 @@ interface OicResourceFoundEvent : Event {
 
 interface OicDeviceFoundEvent : Event {
   OicDeviceInfo device;
+  sequence<OicResource> resources;  // url, resourceType, interfaces
 };
 
 dictionary OicDiscoveryOptions {  // all properties are null by default, meaning “find all”
   USVString? deviceId;
-  USVString? resourceId;
+  USVString? resourceURL;
```

Comment:
We had a round on this with Domenic, and we have to use URL...
See https://w3ctag.github.io/design-principles/